src/search/cartesian_abstractions/utils_landmarks.cc

```diff
@@ -38,10 +38,23 @@ vector<FactPair> get_fact_landmarks(const LandmarkGraph &graph) {
     return facts;
 }
 
-VarToValues get_prev_landmarks(const LandmarkGraph &graph, const FactPair &fact) {
+utils::HashMap<FactPair, LandmarkNode *> get_facts_to_landmarks_map(
```

Comment:
I would rename this to `get_fact_to_landmark_map` to emphasize that the map takes a single fact and maps it to a single landmark.